cri/v1alpha2/cri.go

```diff
@@ -74,6 +74,12 @@ const (
 
 	// networkNotReadyReason is the reason reported when network is not ready.
 	networkNotReadyReason = "NetworkPluginNotReady"
+
+	// passthruKey to specify whether a interface is passthru to qemu
+	passthruKey = "io.alibaba.pouch.runv.passthru"
```

Comment:
`io.alibaba.pouch.runv.passthru` or `io.alibaba.pouch.vm.passthru`？ @Ace-Tang

Reply:
You are right, should be `io.alibaba.pouch.vm.passthru`, could you like help to fix this ? @flyer103